plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/editors/featuremodel/actions/ChangeConstraintDescriptionAction.java

```diff
@@ -0,0 +1,74 @@
+/* FeatureIDE - A Framework for Feature-Oriented Software Development
+ * Copyright (C) 2005-2017  FeatureIDE team, University of Magdeburg, Germany
+ *
+ * This file is part of FeatureIDE.
+ *
+ * FeatureIDE is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FeatureIDE is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with FeatureIDE.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * See http://featureide.cs.ovgu.de/ for further information.
+ */
+package de.ovgu.featureide.fm.ui.editors.featuremodel.actions;
+
+import static de.ovgu.featureide.fm.core.localization.StringTable.CHANGE_DESCRIPTION;
+import static de.ovgu.featureide.fm.core.localization.StringTable.CONSTRAINT_DESCRIPTION;
+import static de.ovgu.featureide.fm.core.localization.StringTable.PLEASE_ENTER_A_DESCRIPTION_FOR_CONSTRAINT_;
+
+import de.ovgu.featureide.fm.core.base.IFeatureModel;
+import de.ovgu.featureide.fm.core.base.event.FeatureIDEEvent;
+import de.ovgu.featureide.fm.core.base.event.FeatureIDEEvent.EventType;
+import de.ovgu.featureide.fm.ui.editors.ChangeFeatureDescriptionDialog;
+
+/**
+ * Opens dialog to change the description of a feature
+ *
+ * @author Marcus Pinnecke (Feature Interface)
+ */
+public class ChangeConstraintDescriptionAction extends SingleSelectionAction {
+
+	public ChangeConstraintDescriptionAction(Object viewer, IFeatureModel featureModel, Object graphicalViewer) {
+		super(CHANGE_DESCRIPTION, viewer);
+	}
+
+	@Override
+	public void run() {
+		String description =
+			"";
+		if (feature.getProperty().getDescription() != null) {
+			description =
+				feature.getProperty().getDescription();
+			description =
+				description.trim();
```

Comment:
Why are you using the old formatter? Did you update to all recent commits from develop before formatting?